requirements/ansible-galaxy.yml

```diff
@@ -38,3 +38,18 @@
   name: onaio.gpg-import
   version: c5bb983d5b4bc5fec82a03d71543fefbe2ddd06e
   scm: git
+
+- src: https://github.com/onaio/ansible-java.git
```

Comment:
@githengi the naming of the role dependencies seems to be off. Should this be `onaio.java` and `onaio.tomcat` instead?